src/Errors.js

```diff
@@ -48,6 +48,9 @@ class Errors {
      */
     get(field) {
         if (this.errors[field]) {
+            if(typeof this.errors[field] === 'string' || this.errors[field] instanceof String) {
```

Comment:
I think we need to keep both of this checks. Here is my test 

![image](https://cloud.githubusercontent.com/assets/2044754/23830378/54524814-072b-11e7-985d-c0abb7a2e43a.png)

I found this check here:  http://stackoverflow.com/a/9436948
And explanation of difference here: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String#Distinction_between_string_primitives_and_String_objects